.travis.yml

```diff
@@ -73,9 +73,17 @@ jobs:
       os: windows
       language: shell
       before_install:
-        - docker build -t cocotb -f Dockerfile.windows .
+        - powershell "Invoke-WebRequest -outfile miniconda3.exe https://repo.continuum.io/miniconda/Miniconda3-latest-Windows-x86_64.exe"
+        - powershell "Start-Process .\miniconda3.exe -ArgumentList '/S /D=C:\miniconda3' -Wait"
+
+        - powershell "Invoke-WebRequest -Uri http://bleyer.org/icarus/iverilog-v11-20190809-x64_setup.exe -OutFile iverilog-x64_setup.exe"
+        - powershell "Start-Process .\iverilog-x64_setup -ArgumentList '/VERYSILENT' -Wait"
+
+        - export CONDA_PATH="\$env:Path = ';C:\miniconda3;C:\miniconda3\Library\mingw-w64\bin;C:\miniconda3\Library\usr\bin;C:\miniconda3\Library\bin;C:\miniconda3\Scripts;C:\miniconda3\bin;C:\miniconda3\condabin;C:\iverilog\bin;' + \$env:Path"
+        - powershell "$CONDA_PATH; conda install --yes -c msys2 m2-base m2-make m2w64-toolchain libpython"
+        - powershell "$CONDA_PATH; pip install tox"
```

Comment:
Can probably skip powershell for these bits with the above suggestion

Reply:
Actually, for whatever reason this does not work (i tried ~10 versions) see: https://travis-ci.org/themperek/cocotb/jobs/645871415

Reply:
Same for the above comment.

Reply:
You have a typo:
```
-export "PATH=$PATH:/C/miniconda3:/C/miniconda3/Library/mingw-w64/bin:/C/miniconda3/Library/usr/bin/C/miniconda3/Library/bin:/C/miniconda3/Scripts:/C/miniconda3/bin:/C/miniconda3/condabin"
+export "PATH=$PATH:/C/miniconda3:/C/miniconda3/Library/mingw-w64/bin:/C/miniconda3/Library/usr/bin:/C/miniconda3/Library/bin:/C/miniconda3/Scripts:/C/miniconda3/bin:/C/miniconda3/condabin"

Reply:
see https://travis-ci.org/themperek/cocotb/jobs/645871415

Reply:
Looks like it works with that fixed: https://travis-ci.org/themperek/cocotb/jobs/646043715

Reply:
Something stuck actually (all this makes me cry)

Reply:
Well, something else breaks. Looks like something very weird is happening, perhaps worth adding `cat /c/Users/travis/.travis/functions` to see what's up

Reply:
It is here: https://travis-ci.org/themperek/cocotb/jobs/646053844

Reply:
I have no idea.

Reply:
This is the failing line, I'll see if I can work out why. Would be nice to report this to travis if we can work it out 
https://travis-ci.org/themperek/cocotb/jobs/646053844#L604

Reply:
Issue is that we added a different `date` to the path, which messes with Travis.

Reply:
https://travis-ci.org/themperek/cocotb/jobs/646084512

Reply:
Nice, that found a bug in the makefile.

Reply:
Not sure if a bug. It works in other cases.

Reply:
Patch incoming - it's a bug in all of our makefiles, we're just never using the broken bits.

Reply:
#1370